docs/security.md

```diff
@@ -0,0 +1,64 @@
+## Security
+
+Metadata application uses Spring OAuth2 and Keycloak authorization to secure REST APIs. W.r.t OAuth2 standards mappings are
+   - Authorization server: Keycloak server
+   - Resource server: Metadata Application
+   - Resource owner: Keycloak users with allowed role
+   - Client: Application calling Metadata REST APIs
+
+Endpoints
+
+   - / and /swagger-ui.html can be accessed by ananymous user. Ex: http://localhost:8085/ and http://localhost:8085/swagger-ui.html
+   - All other endpoints: 
+        - Read(GET) operation can be issued by any authenticated user with keycloak. The user can have any role assigned
```

Comment:
Initial spaces to indent

Reply:
Right now, given that we don't provide access to studies previous to publication, read operations should be anonymous instead of via a read-only user.